CODE-OF-CONDUCT.md

```diff
@@ -1,19 +1,40 @@
-### Contributor Code of Conduct
+# Contributor Code of Conduct
 
-We follow the [CNCF Contributor Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md). Additionally, we commit to the following guidelines as detailed on the [SPIFFE Code of Conduct](https://github.com/spiffe/spiffe/blob/main/CODE-OF-CONDUCT.md):
+We follow the [CNCF Contributor Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+Additionally, we commit to the following guidelines as detailed on the
+[SPIFFE Code of Conduct](https://github.com/spiffe/spiffe/blob/main/CODE-OF-CONDUCT.md):
 
-### Community Guidelines
+## Community Guidelines
```

Comment:
I think this was intended to be at the same heading level as the Contributor Guidelines

Reply:
the issue here is that linter enforce to have a single top level  heading `#`, so we can:

1- add a new heading and keep all existing headers as `##`
2- add a comment to bypass this file for that validation